### DISCUSSION TITLE
How to transcode album?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#1008] [Q&A]
I have an album I want to transcode. The only transcode option I see in the playlist name context menu `transcode all`.
I don't see such option for the album or selected song(s)

Should I select output folder? The button does nothing, maybe flatpak permissions? 
Does `Sync` setting relevant to transcode?

**Comment by Taiko2k** [comment]
Try have a read of the page here: https://github.com/Taiko2k/TauonMusicBox/wiki/Transcoding-audio-files

The option should appear at the bottom the the track context menu. Don't worry about sync, that a related but different feature.

**Reply by Solarunit** [reply]
hmm, I don't see such an option

![](https://i.imgur.com/hHVVxbT.png)
